### ISSUE TITLE
AttributeError on Protocol.__str__()

### ISSUE THREAD

**Issue by SamCB** [ecederstrand/exchangelib#10]
When trying to print a `Protocol` object, I get:

```
Traceback (most recent call last):
  File "main.py", line 35, in <module>
    print(account.protocol)
  File "/usr/exchangelib/exchangelib/protocol.py", line 163, in __str__
    self.version.shortname,
AttributeError: 'Version' object has no attribute 'shortname'
```

Deleting that line (as well as relevant line in the string) raises another error:

```
Traceback (most recent call last):
  File "main.py", line 35, in <module>
    print(account.protocol)
  File "/usr/exchangelib/exchangelib/protocol.py", line 164, in __str__
    self.version.name,
AttributeError: 'Version' object has no attribute 'name'
```

I switched that line to `self.version.fullname,` and it worked. That's the state of my Fork that may be PR'd some time when I've done a few extra things we've needed.

**Comment by SamCB** [comment]
Decided that I might as well chuck the fix up now anyway.

**Comment by ecederstrand** [comment]
Fixed by d9ec4051ab9d50f0dc56f9cfda7f05d57fe2472d
